src/formio/components/FileField.stories.mdx

```diff
@@ -0,0 +1,124 @@
+import {Canvas, Meta, Props, Story} from '@storybook/addon-docs';
+
+import {FileField} from './FileField';
+import {FormDecorator, FormioComponent} from './story-util';
+import {
+  BASE_URL,
+  ENDPOINT,
+  mockFileUploadError,
+  mockFileUploadPost,
+} from '/src/mocks/FileUploadMocks';
+
+export const FILETYPES = [
+  'all',
+  'heic',
+  'png',
+  'jpg',
+  'pdf',
+  'xls',
+  'xlsx',
+  'csv',
+  'txt',
+  'doc',
+  'docx',
+  'open_office',
+  'zip',
+  'rar',
+  'tar',
+  'msg',
+  'dwg',
+];
+
+<Meta
+  title="Form.io components/FileUpload"
+  component={FileField}
+  argTypes={{
+    id: {
+      type: {name: 'string'},
+    },
+    label: {
+      type: {name: 'string', required: false},
+    },
+    fileTypes: {
+      options: FILETYPES,
+      control: {type: 'check'},
+    },
+    uploadMode: {
+      options: ['single', 'multiple'],
+      control: {type: 'radio'},
+    },
+  }}
+  decorators={[FormDecorator]}
+/>
+
+# File Upload
+
+export const Template = ({
+  componentUrl,
+  description = '',
+  id = '',
+  label = '',
+  setStorage,
+  ...args
+}) => (
+  <FormioComponent
+    component={{
+      key: 'bestand',
+      type: 'file',
+      input: true,
+      label,
+      description,
+      storage: setStorage,
+      url: componentUrl,
+    }}
+  />
+);
+
+## Happy flow
+
+<Canvas>
+  <Story
+    name="Happy flow"
+    args={{
+      componentUrl: `${BASE_URL}${ENDPOINT}`,
+      description: 'Klik om een bestand te selecteren en in te zenden',
+      id: 'option',
+      label: 'Bestand meesturen',
+      setStorage: 'url',
+    }}
+    parameters={{
+      controls: {sort: 'requiredFirst'},
+      msw: {
+        handlers: [mockFileUploadPost],
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+<Props story="Happy flow" />
+
+## Error flow
+
+<Canvas>
+  <Story
+    name="Error flow"
+    args={{
+      componentUrl: `${BASE_URL}${ENDPOINT}`,
+      description: 'Klik om een bestand te selecteren en in te zenden',
+      id: 'option',
```

Comment:
this maps to the component `key` of the formio component, so I'd use something more appropriate like `fileUpload` here